src/plugins/orangehrmRecruitmentPlugin/Dao/VacancyDao.php

```diff
@@ -211,4 +203,27 @@ public function getPublishedVacancyList(): array
         $qb->addOrderBy('vacancy.updatedTime', ListSorter::DESCENDING);
         return $qb->getQuery()->execute();
     }
+
+    /**
+     * @param int $jobTitleId
+     * @return JobTitle|null
+     */
+    public function isActiveJobVacancy(int $jobTitleId): ?JobTitle
+    {
+        return $this->getRepository(JobTitle::class)->findOneBy(['id' => $jobTitleId, 'isDeleted' => false]);
+    }
+
+    /**
+     * @param int $hiringManagerId
+     * @return Employee|null
+     */
+    public function isActiveHiringManger(int $hiringManagerId): ?Employee
```

Comment:
here also return type should be boolean according to the method name
```php
        return $this->getRepository(Employee::class)->findOneBy(
            [
                'empNumber' => $hiringManagerId,
                'employeeTerminationRecord' => null
            ]
        ) instanceof Employee;
```

Reply:
fixed